style.css

```diff
@@ -0,0 +1,16 @@
+#nav-bar{
+    position: fixed;
+    top: 0;
+    }
+    
+    #header{
+        display: flex;
+        align-items: center;
+        justify-content: space-around;
+    }
```

Comment:
Great using flex-box for putting the items in place. I will suggest giving some margin to the buttons. If your previous intention with this css code, was to do so, yous should apply display-flex in your actual container.
a hint; your current code is giving space-between image and nav; not between buttons!

Reply:
Hi, @HFG43!! 👋
- Well done on making inline comments with suggestions and reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use `nav` nested of your `header`". 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.

- [ ] ***[OPTIONAL]*** I can see some grammatical errors such as you have used, `I will suggest`, which would be `I  would suggest`, and `yous` would be `you`. Kindly use [Grammarly](https://app.grammarly.com/) so that there are no spelling mistakes and other grammatical mistakes. It is more professional to check the spelling while giving any suggestion in every aspect of our job and profession.